src/org/zaproxy/zap/extension/ascanrules/TestSQLInjection.java

```diff
@@ -1451,6 +1489,38 @@ public static String getURLDecode(String msg) {
 		return result;
 	}
 
+	/**
+	* add two numbers with Arithmetic Overflow check
+	* @param firstNumber
+	* @param secondNumber
+	* @return
+	*/
+	public static int addWithOverflowCheck(int firstNumber, int secondNumber) {
```

Comment:
- Should be private;
 - Could you add a TODO comment mentioning that this should be replaced with the corresponding `Math` method when targeting Java 8?